PlayerController.cs

```diff
@@ -18,19 +23,19 @@ void Update()
         
     }
 
-    [Header("General Setup Settings")]
-    [SerializeField] private InputAction movement;
-    [Tooltip("How fast player moves up and down based upon player input")][SerializeField] float controlSpeed = 30f;
-    [Tooltip("How far player moves horizontally")][SerializeField] float xRange = 10f;
-    [Tooltip("How far player moves vertically")][SerializeField] float yRange = 10f;
+    [Header("General Setup Settings")] // UI header title
+    [SerializeField] private InputAction movement; // displays and makes editable this variable Input Action
```

Comment:
This defines Unity's new [Input Management System](https://docs.unity3d.com/Packages/com.unity.inputsystem@1.7/manual/index.html)